raisinbread/README.md

```diff
@@ -9,7 +9,19 @@ adjusted when existing features are changed or removed.
 The sections below provide some insights into installing, modifying and exporting 
 the dataset.
 
-### Installing RB
+### Automated RB Installation
+The script `tools/raisinbread_refresh.php` includes functionality that drops all existing tables in the
+database and sources all of the RaisinBread data automatically. The script will preserve database settings such as API Keys and server configuration settings to simplify the process of switching between development environments. This tool can be run through the provided Makefile. To run this script, navigate to the LORIS
+root directory and run the following command:
+
+```bash
+make testdata
+```
+
+If RaisinBread is being installed for the first time, the steps outlined below in the 
+[Configuring](https://github.com/aces/Loris/tree/23.0-release/raisinbread#configuring) section must be completed. 
```

Comment:
It might be better to not use a hyperlink here because we'll have to update the version info in the link every release.